civis/response.py

```diff
@@ -3,6 +3,9 @@
 from civis._utils import camel_to_snake
 
 
+_RETURN_TYPES = frozenset({'snake', 'raw', 'pandas'})
```

Comment:
What are the snake and raw return types? I couldn't figure out what this is. Thanks Jackson!

Reply:
It's in the docstring of `APICilent`:

https://github.com/civisanalytics/civis-python/blob/6c7213ec0bd8b8e42203540597c001f6f530a7ef/civis/civis.py#L352-L361

The default value in `APIClient` is `"snake"`, since in the Python world you'd expect the attributes to be in `snake_case`. If you pick `"raw"` (which I've never used myself in practice), you'd get attributes in `camelCase` instead.